lib/Common/Memory/Recycler.cpp

```diff
@@ -937,6 +937,18 @@ Recycler::SetIsInScript(bool isInScript)
     this->isInScript = isInScript;
 }
 
+bool
+Recycler::HasNativeGCHost() const
+{
+    return this->hasNativeGCHost;
```

Comment:
should we initialize in the constructor for completeness?

Reply:
Yes.